platform/core/src/capabilities/PersistenceCapability.js

```diff
@@ -104,14 +104,16 @@ define(
          * persistence.
          */
         function notifyOnError(error, domainObject, notificationService, $q){
-            var errorMessage = "Unable to persist " + domainObject.getModel().name;
+            var errorMessage = "Unable to persist " + domainObject.getModel().name,
+                notification;
             if (error) {
                 errorMessage += ": " + formatError(error);
             }
 
-            notificationService.error({
+            notification = notificationService.error({
```

Comment:
Looks like a dead store. Appears the intent here was to use this `notification` later; is there additional behavior that's meant to be implemented here? (If not, think we should just remove this variable)